### DISCUSSION TITLE
Symbols and non-word characters

### DISCUSSION THREAD

**Discussion by JuanFF** [openai/whisper#242] [Q&A] [answered]
Hello,
I see a lot of special characters (e.g. `>>`) and annotations between brackets (e.g. `[MUSIC]`) in the transcripts. Is there any key with explanations on what these symbols are used for?
Thanks!

**Comment by jianfch** [comment, marked as answer]
It's likely the a way of mapping the speech that the model picked up from the raw text in transcripts of the training data which received no "significant standardization" as described in the paper.

**Reply by jongwook** [reply]
This is correct; we fine-tuned the model to make it less likely to produce those, but it may still sample those non-speech annotations. We have a mechanism to prevent sampling those tokens by default:

https://github.com/openai/whisper/blob/9e653bd0ea0f1e9493cb4939733e9de249493cfb/whisper/tokenizer.py#L235-L270
